cve_bin_tool/checkers/xml2.py

```diff
@@ -60,6 +60,9 @@ def guess_xml2_version(lines):
                 new_guess2 = match.group(1).strip()
                 if len(new_guess2) > len(new_guess):
                     new_guess = new_guess2
+        # If no version guessed, set version to UNKNOWN
+        if new_guess == "":
+            new_guess = "UNKNOWN"
         return new_guess
```

Comment:
I think this can be somewhat simplified without losing clarity
```suggestion
        return new_guess or "UNKNOWN"
```